src/main/java/org/openstreetmap/atlas/geography/atlas/items/LocationItem.java

```diff
@@ -66,6 +76,9 @@ public LocationIterableProperties toGeoJsonBuildingBlock()
         tags.put("osmIdentifier", String.valueOf(getOsmIdentifier()));
         tags.put("itemType", String.valueOf(getType()));
 
+        final Optional<String> shardName = getAtlas().metaData().getShardName();
```

Comment:
Why is this done in the `bounds()` call? Doesn't seem consistent with the other classes.

Reply:
This was an addition I added to the legacy GeoJSON code. Taking this out, since it doesn't really matter for this PR.